onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -146,28 +163,37 @@ def quantize_data(data, quantize_range, qType, symmetric=True):
             S: scale
             z: zero point
     '''
-    rmin = min(min(data), 0.)
-    rmax = max(max(data), 0.)
```

Comment:
we add this re-alignment to avoid issues like padding in Conv is 0. Without this alignment, padding == 0 will not be mapped correctly. 

Any reason to remove it?

Reply:
This alignment is happening in `compute_scale_zp` now

Reply:
Nice!